src/utils/file.js

```diff
@@ -16,15 +23,30 @@ export function createPackageJson(config) {
   packageJson.description = config.description;
   packageJson.authors = author;
   packageJson.license = config.license;
+  packageJson.dependencies = dependencies;
 
-  const json = JSON.stringify(packageJson, null, '\t');
+  const json = JSON.stringify(packageJson, null, '  ');
 
   fs.writeFileSync(`${config.projectName}/package.json`, json);
 }
 
 /**
  * Generate a simple index.js file
  */
-export function createIndexFile(folderName) {
-  fs.writeFileSync(`${folderName}/src/index.js`, constants.indexJS.fileContents);
+export function createIndexFile(folderName, usingExpress) {
+  if (usingExpress) {
+    fs.writeFileSync(
```

Comment:
We should put this in a new file with other express-related logic, so when we do have to change the code to fit a new version, it is all in one place.